srtp/srtp.c

```diff
@@ -3545,6 +3546,7 @@ static srtp_err_status_t srtp_protect_rtcp_aead(
      */
     /* Note: This would need to change for optional mikey data */
     auth_tag = (uint8_t *)hdr + *pkt_octet_len;
+    octet_string_set_to_zero(auth_tag, tag_len);
```

Comment:
This looks like a latent bug.  Have you checked?

Reply:
I'm not sure why that line is there; the tests pass without it.  I've deleted it.